tests/test.mk

```diff
@@ -23,6 +23,7 @@ include tests/soter/soter.mk
 include tests/tools/tools.mk
 include tests/themis/themis.mk
 include tests/themispp/themispp.mk
+include tests/themispp_simple/themispp_simple.mk
```

Comment:
Now I remember that RustThemis has some special relationship with included Makefiles, but this should be okay because **tests/test.mk** is not included when building by Cargo:

https://github.com/cossacklabs/themis/blob/a558d85896419f97898ea22e384637f5467204c5/Makefile#L363-L370